Cargo.toml

```diff
@@ -54,7 +52,7 @@ version = "1.5.0"
 optional = true
 
 [dependencies.rustls]
-version = "0.17.0"
+version = "0.16.0"
```

Comment:
unfortunately, this is required for compatibility with `async-tls`. They have an [issue](https://github.com/async-rs/async-tls/issues/20) tracking the update to fix this.

Reply:
as discussed offline, we'll just keep using tokio-rustls.